extensions/data-plane/data-plane-azure-storage/src/main/java/org/eclipse/edc/connector/dataplane/azure/storage/pipeline/AzureStorageDataSinkFactory.java

```diff
@@ -66,10 +69,15 @@ public boolean canHandle(DataFlowRequest request) {
     @Override
     public @NotNull Result<Void> validateRequest(DataFlowRequest request) {
         var dataAddress = request.getDestinationDataAddress();
+        var dataSourceAddress = request.getSourceDataAddress();
+
         try {
             validateAccountName(dataAddress.getStringProperty(ACCOUNT_NAME));
             validateContainerName(dataAddress.getStringProperty(CONTAINER_NAME));
             validateKeyName(dataAddress.getKeyName());
+            if (dataSourceAddress.hasProperty(BLOB_PREFIX)) {
```

Comment:
is this really a problem? couldn't we just log a warning and ignore the BLOB_NAME?